### ISSUE TITLE
invalid field path error

### ISSUE THREAD

**Issue by jonathan-chin** [AntlerVC/firetable#212]
I was playing around and adding some new columns. I think I hit 'Create' without giving the column an id / name. shortly after, I get this error:

`FirebaseError: Invalid field path (columns.``.config). Paths must not be empty, begin with '.', end with '.', or contain '..'`

I now can't do anything on the app because this error halts everything.

edit: I deleted the firetable project on my laptop and re-inited a new one. I get the same error.

**Comment by shamsmosowi** [comment]
We'll add validation to stop that from happening, but for now you can go to the firestore console and delete the fault column, you should be able to find it under ```_FIRETABLE_/settings/schema/[tableName]```

**Comment by jonathan-chin** [comment]
> We'll add validation to stop that from happening, but for now you can go to the firestore console and delete the fault column, you should be able to find it under `_FIRETABLE_/settings/schema/[tableName]`

Hi, @shamsmosowi I just accidentally triggered this again. I'm looking under `_FIRETABLE_/settings/schema/[tableName]/columns` and I see all the named columns but not the faulty column. it seems `columns` is a key-value object where key is the name of the column. but if I didn't name the column, there would be no key.

is this info stored somewhere else?

**Comment by shamsmosowi** [comment]
actually it looks like a firebase SDK issue. you'll just need to clear out the application data, by opening to console,  going to the application tab, and clicking the "clear site data" button
![image](https://user-images.githubusercontent.com/13803906/93942244-62455a00-fd73-11ea-9b5e-84c42ba8f077.png)

**Comment by jonathan-chin** [comment]
@shamsmosowi ok, so that worked. however, my original table now doesn't show up anymore, even though the data and schema are showing up in firestore.

I tested it again: created a new table (it worked fine), intentionally created a faulty column, did the `clear site data` fix. the new table I created is still there but the original table still doesn't show up. both tables have data in firestore.

thoughts?
